eframe/Cargo.toml

```diff
@@ -42,12 +42,16 @@ image = { version = "0.23", default-features = false, features = ["png"] }
 rfd = "0.6"
 
 [features]
-default = ["default_fonts", "egui_glow"]
+default = ["default_fonts", "egui_glow", "clipboard", "links"]
 
 # If set, egui will use `include_bytes!` to bundle some fonts.
 # If you plan on specifying your own fonts you may disable this feature.
 default_fonts = ["egui/default_fonts"]
 
+clipboard = ["egui_glium/clipboard", "egui_glow/clipboard"]
+
+links = ["egui_glium/links", "egui_glow/links"]
+
 # Enable saving app state to disk.
 persistence = [
   # we cannot touch egui_glium or egui_glow here due to https://github.com/rust-lang/cargo/issues/8832
```

Comment:
If you follow the above link you'll see why this PR causes a problem. Anyone using `eframe` now would need to compile both `egui_glium` AND `egui_glow`, which is not so good.

I'm not sure what a good solution is to this. I find feature and the cargo dependency system quite confusing.

Reply:
Ooof, I didn't know about that, thanks for the link. At least it looks like this will be fixed at some point in the future.

The only thing I could think of right now is adding a feature for each backend (e.g.: `egui_glow_clipboard` and `egui_glium_clipboard`, etc.), but I'm not happy with that.

It would solve the issue I'm having and I'll be happy to update the PR with that, but it makes for an unpleasant "cargo API". I totally understand if you don't want to go that route.

Reply:
I'm hoping that `egui_glow` can take over as the default backend, and that we can remove `egui_glow` support in the future, so in a few months we will hopefully only have `egui_glow` in this file, and the problem will be solved that way.

Reply:
In the meantime you can bypass `eframe` and just use `egui_glow` and `egui_web` directly. `eframe` is an extremely small crate – this is all the code in it: https://github.com/emilk/egui/blob/master/eframe/src/lib.rs

Reply:
Great suggestion, thanks!